frontend/components/form/checkbox/FormCheckbox.vue

```diff
@@ -2,7 +2,7 @@
   <div class="relative flex flex-row items-center justify-start">
     <input
       :id="uuid"
-      class="cursor-pointer mb-0 peer appearance-none w-[1.375rem] h-[1.375rem] border border-light-menu-selection rounded-sm bg-light-button dark:bg-dark-button dark:border-dark-menu-selection focus-brand"
+      class="connect cursor-pointer mb-0 peer appearance-none w-[1.375rem] h-[1.375rem] border border-light-menu-selection rounded-sm bg-light-button dark:bg-dark-button dark:border-dark-menu-selection focus-brand"
```

Comment:
Would it make sense to apply this to an element on the page itself rather than a component as if there are say social accounts then they'll be on in between?

Reply:
I understand what you mean, I would still need to research whether there's another way to implement this, as the current one doesn't seem to be a good practice. I'll go ahead and start with your direction on the element rather than the component and see how it goes.